lua/conform/formatters/sqlfluff.lua

```diff
@@ -0,0 +1,20 @@
+local util = require("conform.util")
+
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/sqlfluff/sqlfluff",
+    description = "A modular SQL linter and auto-formatter with support for multiple dialects and templated code.",
+  },
+  command = "sqlfluff",
+  args = { "fix", "--force", "--dialect=ansi", "-" },
+  stdin = true,
+  cwd = util.root_file({
+    ".sqlfluff",
+    "pep8.ini",
+    "pyproject.toml",
+    "setup.cfg",
+    "tox.ini",
```

Comment:
These all look python-specific. Is there a reason to include them as root files for sqlfluff?

Reply:
https://docs.sqlfluff.com/en/stable/configuration.html